README.md

```diff
@@ -87,28 +87,27 @@ This is an static landing page and so, I (cynthia, who is writing this nice expl
 ---
 
 #CONTRIBUTE
-## Do you wanna contribute providing translation for the new openSUSE website?
+## Do you want to contribute by providing translations for the new openSUSE website?
```

Comment:
@ddemaio Since you made most of the text here, I prefer you give it a LGTM or say it if you prefer to keep the old text.